site/specs-source/webRtc.json

```diff
@@ -499,7 +499,7 @@
                }
             ],
             "requestBody": {
-               "description": "Subscriptions the participant should be created with",
+               "description": "Optional set of subscriptions to set on the participant.\nCalling this endpoint with no/empty body will only add the participant to the session.\n\nNOTE: the request body for this endpoint is OPTIONAL and provided as a convenience to avoid calls to this and the Update Participant Subscriptions endpoint.\n\nSubscriptions the participant should be created with.",
```

Comment:
I think that no body is permitted, but that an empty body is not allowed.  I believe that there does not need to be a body, but if there is a body it must contain the sessionId element.   You could test with Postman to confirm / refute my allegation .

Reply:
Just tested this, and if you call the endpoint with an empty object `{}`, it will still add the participant to the session and return a 204. If you call the endpoint with the following body:

```
{
    "sessionId": "",
    "participants": [
        {
            "participantId": "",
        }
    ]
}
```
That results in a 500 error. I can change this to "Calling this endpoint with no body", or leave it as is -- up to you!

Reply:
Do we still need the old text, "Subscriptions the participant should be created with"? I vote that we remove it.

Reply:
I agree, it seems out of place now with the other additions. I mostly kept it in there as a legacy kind of thing

Reply:
Went ahead and removed it, think it makes sense without it